hw2/briansp8210/q2/test_class.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+import _vector
+import math
+
+class TestClass:
+    def helper(self, v1, v2, deg):
```

Comment:
I feel that `equal_helper` will make sense.

Reply:
That makes sense, thanks.